modules/AOR_Reports/AOR_Report.php

```diff
@@ -1560,17 +1560,17 @@ function build_report_query_where($query = array())
                                 $params = base64_decode($condition->value);
                             }
                             $date = getPeriodEndDate($params)->format('Y-m-d H:i:s');
-                            $value = '"' . getPeriodDate($params)->format('Y-m-d H:i:s') . '"';
+                            $value = '\'' . getPeriodDate($params)->format('Y-m-d H:i:s') . '\'';
```

Comment:
Why escaping and not just switch quotes like:
 `$value = "'" . getPeriodDate($params)->format('Y-m-d H:i:s') . "'";`